release notes/v0.48.0.md

```diff
@@ -38,8 +38,8 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 
 ## Maintenance and internal improvements
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#2770` Refactors parts of the JS module._
+- [browser#1043](https://github.com/grafana/xk6-browser/pull/1043) Refine tests.
+- [browser#1047](https://github.com/grafana/xk6-browser/pull/1047) Update dependencies.
```

Comment:
Hey @ankur22, if we want to add them to the release notes then I would prefer we should detail a bit more providing information that users should be aware of or about _why we did_.

Reply:
These are internal changes that aren't very useful to the user, but we usually [list](https://github.com/grafana/k6/blob/5ceb210056edc52cd64da4c17d860c868487b268/release%20notes/v0.47.0.md#L110-L112) them still. So i don't think we need to add many more details here. WDYT?

Reply:
I'm of the opinion now that we are doing core+browser we should condense a bit more the various information. Anyway, I think we may discuss this on the final release notes PR. I'm pretty sure any individual contributor has their own opinion about this.